cdn_lambda/functions/origin_response/origin_response.py

```diff
@@ -0,0 +1,63 @@
+import json
+import logging
+import re
+
+LOG = logging.getLogger("origin-response")
+
+
+class LambdaClient(object):
+    def __init__(self, conf_file="lambda_conf.json"):
```

Comment:
This seems broken:

- origin_request uses lambda_config.json
- origin_response uses lambda_conf.json

So you would have to deploy two copies of the config file under different names for this to work - doesn't seem intentional.